mysite/blog/templates/blog/home.html

```diff
@@ -54,4 +61,30 @@ <h2><a class="article-title" href="{% url 'post-detail' post.slug %}">{{ post.ti
         <p class="article-content">{{ post.content|truncatechars:300 }}</p>
     </div>
 </article>
-{% endfor %} {% endblock%}
+
+{% endfor %}
+
+<div class="pagination">
+
+    {% if posts.has_previous %}
+    <a class="btn btn-outline-info mb-4" href="?page=1">First</a>
+    <a class="btn btn-outline-info mb-4" href="?page={{ posts.previous_page_number }}">Previous</a>
+    {% endif %}
+
+    {% for num in posts.paginator.page_range %}
+    {% if posts.number == num %}
+    <a class="btn btn-info mb-4" href="?page={{ num }}">{{ num }}</a>
+    {% elif num > posts.number|add:'-3' and num < posts.number|add:'3' %}
```

Comment:
Hey @kritikaparmar-programmer let's keep 2 blogs per page.

Reply:
As you say ...😁

Reply:
Done the changes..